x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/datafeed/DatafeedConfig.java

```diff
@@ -286,6 +291,12 @@ public Integer getScrollSize() {
         return scrollSize;
     }
 
+    public Optional<Tuple<Version, String>> minRequiredClusterVersion() {
```

Comment:
Just FYI: In my PR introducing remote cluster version validation, I decided to replace `TransformConfig.minRequiredClusterVersion` with `TransformConfig.getAdditionalValidations` the reason being code duplication.